mesheryctl/internal/cli/root/registry/update.go

```diff
@@ -55,6 +55,12 @@ mesheryctl registry update --spreadsheet-id 1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdw
 mesheryctl registry update --spreadsheet-id 1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwizOJmeMw --spreadsheet-cred $CRED --model "[model-name]"
 	`,
 	PreRunE: func(cmd *cobra.Command, args []string) error {
+		spreadsheetID, _ := cmd.Flags().GetString("spreadsheet-id")
+		spreadsheetCred, _ := cmd.Flags().GetString("spreadsheet-cred")
+
+		if spreadsheetID == "" || spreadsheetCred == "" {
+			return cmd.Help()
```

Comment:
Should be a Meshkit Error

Reply:
@lekaf974 you mean, wrap the error with the error from the "github.com/layer5io/meshkit/errors" package ?

Reply:
yep this is the way we want to standardized error in mesheryctl